src/aura/CRLP_Rollup/CRLP_Rollup.cmp

```diff
@@ -0,0 +1,286 @@
+    <aura:handler name="change" value="{!v.activeRollup.Operation__c}" action="{!c.onChangeOperation}"/>
+    <aura:handler name="change" value="{!v.activeRollup.Yearly_Operation_Type__c}"
+                  action="{!c.onChangeYearlyOperation}"/>
+
+    <!--activeRollup:<br/>
+    Detail Object API Name:    {!v.activeRollup.Detail_Object__r.QualifiedApiName}<br/>
+    Detail Field API Name:    {!v.activeRollup.Detail_Object__r.Label}<br/>
+    Summary Object API Name: {!v.activeRollup.Summary_Object__r.QualifiedApiName}<br/>
+    Summary Field API Name: {!v.activeRollup.Summary_Field__r.QualifiedApiName}<br/>
+    <br/>
+    cachedRollup:<br/>
+    Detail Object API Name:    {!v.cachedRollup.Detail_Object__r.QualifiedApiName}<br/>
+    Detail Field API Name:    {!v.cachedRollup.Detail_Field__r.QualifiedApiName}<br/>
+    Summary Object API Name: {!v.cachedRollup.Summary_Object__r.QualifiedApiName}<br/>
+    Summary Field API Name: {!v.cachedRollup.Summary_Field__r.QualifiedApiName}<br/>
+-->
+    <!--yearly Op Label: {!v.selectedYearlyOperationLabel}<br/>
+    yearly Op : {!v.activeRollup.Yearly_Operation_Type__c}<br/>
+    Op Label: {!v.selectedOperationLabel}<br/>
+    Op : {!v.activeRollup.Operation__c}-->
+
+    <!-- markup -->
+
+    <!--BEGIN NEW ROLLUP-->
+    <aura:if isTrue="{!v.mode == 'create'}">
+        <c:CRLP_CreateRollup labels="{!v.labels}" mode="{!v.mode}" selectedTemplate="{!v.selectedTemplate}"/>
+
+        <aura:set attribute="else">
+
+            <!--END NEW ROLLUP-->
+            <!--START FORM-->
+            <div aria-labelledby="rollupdetail">
+                <form class="slds-form_stacked">
+                    <lightning:layout>
+                        <lightning:layoutItem size="12" largeDeviceSize="8">
+
+                            <lightning:layout multipleRows="true" verticalAlign="start">
+                                <lightning:layoutItem padding="around-medium" smallDeviceSize="6" size="12">
+                                    <lightning:input aura:id="nameInput"
+                                                     label="{!v.labels.rollupLabel + v.labels.rollupName}"
+                                                     name="{!v.labels.rollupLabel + v.labels.rollupName}"
+                                                     value="{!v.activeRollup.MasterLabel}"
+                                                     required="{! !v.isReadOnly}"
+                                                     readonly="{!v.isReadOnly}"/>
+                                </lightning:layoutItem>
+                                <lightning:layoutItem padding="around-medium" smallDeviceSize="6" size="12">
+                                    <aura:if isTrue="{!v.isReadOnly}">
+                                        <!--todo: how can this be screen-readable?-->
+                                        <ui:outputCheckbox value="{!v.activeRollup.Active__c}" class="rollupCheckbox"
```

Comment:
This is a good question that we'll need to be aware of when we arrive at the CMT creation story. For now, all we're doing is working with a JSON object to keep the data organized, which we convert when we do our retrieval from Apex. We're hopeful that will be an easy way to then take that string and work with it in Apex, but this could require some refactoring if we find different with those updates.